templates/index.html

```diff
@@ -93,11 +98,11 @@
                     <div id="vertical-line"></div>
                     <div id="info-box-Right">
                         <div class="label" id="label-weight">Weight</div>
-                            <label class="paragraph" id="paragraphWeight">0</label><div></div>
+                            <label class="paragraph" id="paragraphWeight">{{ animal.weight }}</label><div></div>
                         <div class="label" id="label-status">Status</div>
-                            <label class="paragraph" id="paragraphStatus">Adopted</label><div></div>
+                            <label class="paragraph" id="paragraphStatus">{{ animal.status }}</label><div></div>
                         <div class="label" id="label-sex">Sex</div>
-                            <label class="paragraph" id="paragraphlSex">Female</label><div></div>                        
+                            <label class="paragraph" id="paragraphlSex">{{ animal.status }}</label><div></div>                        
```

Comment:
Status to płeć?

Reply:
Nope, that was a mistake, fixed that in the newest commit.